modules/codelite/codelite_project.lua

```diff
@@ -111,7 +111,7 @@
 
 		-- TODO: ...
 	end
-
+  
```

Comment:
This can probably be reverted.

Reply:
Adding the extra line here actually conforms to the rest of the file so I will leave it.